.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: false
+cache: pip
```

Comment:
Please, use the [cache with directories](https://github.com/OCA/maintainer-quality-tools/blob/620b87395086ba546c50339601eb0f21236cc3a1/sample_files/.travis.yml#L3-L6)